funcbench/manifests/benchmark/2_job.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: funcbench-test-{{ .PR_NUMBER }}
+  namespace: funcbench-{{ .PR_NUMBER }}
+spec:
+  # never re-create a new pod
+  backoffLimit: 0
+  template:
+    spec:
+      restartPolicy: Never
+      containers:
+      - name: funcbench
+        image: nevill/funcbench:20200324
+        imagePullPolicy: Always
+        args:
+          - "-v"
+          - "-o"
+          - "{{ .GITHUB_ORG }}"
+          - "-r"
+          - "{{ .GITHUB_REPO }}"
+          - "-p"
+          - "{{ .PR_NUMBER }}"
+          - "-w"
+          - "/build"
+          - "{{ .BRANCH }}"
+          - "{{ .REGEX }}"
+        env:
+          - name: GITHUB_TOKEN
+            value: "{{ .GITHUB_TOKEN }}"
+          - name: GITHUB_SHA
```

Comment:
I am removing the use of `GITHUB_SHA` in an upcoming PR as it is not of any use inside funcbench, we can either wait for it to get merged or merge this before.

Reply:
Either way is fine. Just to mention that, `GITHUB_SHA` is still used in [funcbench](https://github.com/prometheus/test-infra/blob/master/funcbench/env.go#L129) now.